src/core/1/plugins/plugins/events/wildcard-events/index.md

```diff
@@ -0,0 +1,86 @@
+---
+type: page
+code: false
+title: Wildcard Events
+order: 250
+---
+
+# Wildcard Events
+
+Kuzzle allows you to listen to multiple events with a single listener thanks to wildcards (`*`).
+
+---
+
+## Single wildcard
+
+| Arguments | Type                                                           | Description                |
+| --------- | -------------------------------------------------------------- | -------------------------- |
+| `request` | <pre><a href=/plugins/1/constructors/request>Request</a></pre> | The normalized API request |
```

Comment:
The payload of an event depends of its type.

For instance, `core:overload` does not provide a Request, but a number